src/lib/libcanvas.js

```diff
@@ -66,6 +76,23 @@ var LibraryCanvas = {
         name = Pointer_stringify(name);
         context[name] = value;
     },
+    lcContextGetPropertyString: function(context, name) {
+        context = Canvas.contextList[context];
+        name = Pointer_stringify(name);
+        var stringBuffer = Canvas.stringBuffer;
+        Module.stringToUTF8(context[name], stringBuffer, 512);
```

Comment:
Does this leaks? if so, how can we manage it?

Reply:
It would leak if it was a new stringBuffer, but that's why I use a unique one Canvas.stringBuffer.

The downside is it has to be evaluated directly and not stored in C